src/email/export.directive.spec.js

```diff
@@ -0,0 +1,28 @@
+describe('Email export directive', function() {
+    var element, compile, scope;
+
+    beforeEach(module('motech-common'));
+    beforeEach(inject(function($compile, $rootScope){
+        compile = $compile;
+        scope = $rootScope.$new();
+        scope.range = 5;
+        scope.month = 10;
+        element = angular.element('<email-export close="hideExportModal()"></email-export>');
+        compile(element)(scope);
+        scope.$digest();
+    }));
+
+    it('should pass close', function () {
+        expect(element.attr('close')).toBeDefined();
+        expect(element.attr('close')).toEqual('hideExportModal()');
+    });
+
+    it('should have tagName element with good value', function () {
```

Comment:
@DamianNowakSoldevelo Change the name of the test to: `'should have tagName with value "EMAIL-EXPORT"'`